plugins/base_commands.py

```diff
@@ -8,8 +8,12 @@
 
 import datetime
 
+import logging
 from spockbot.plugins.base import PluginBase
 from spockbot.vector import Vector3
+import json
+tpareqs = {}
```

Comment:
Nah it should be an instanced variable set up in init. No need to share between objects. This code will explode hard if you ever tore down the plugin and then recovered (same for class scope). 

That's maybe a bit of an esoteric example but correct scoping is a super important programming instinct to learn. Code smells and all that